src/bitcoin-cli.cpp

```diff
@@ -70,7 +70,7 @@ static void SetupCliArgs(ArgsManager& argsman)
     argsman.AddArg("-conf=<file>", strprintf("Specify configuration file. Relative paths will be prefixed by datadir location. (default: %s)", BITCOIN_CONF_FILENAME), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     argsman.AddArg("-datadir=<dir>", "Specify data directory", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     argsman.AddArg("-generate", strprintf("Generate blocks immediately, equivalent to RPC getnewaddress followed by RPC generatetoaddress. Optional positional integer arguments are number of blocks to generate (default: %s) and maximum iterations to try (default: %s), equivalent to RPC generatetoaddress nblocks and maxtries arguments. Example: bitcoin-cli -generate 4 1000", DEFAULT_NBLOCKS, DEFAULT_MAX_TRIES), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
-    argsman.AddArg("-addrinfo", "Get the number of addresses known to the node, per network and total.", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-addrinfo", "Get the number of addresses known to the node, per network and total. These are filtered for quality, so the total may be lower than the numbers in the debug log.", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
Not sure if it's good practice to refer to the debug log in RPC helps. How about something like "Get the quality-filtered number of addresses known to the node, per network and total. The total number of addresses known to the node may be higher."
(and something similar below)

Reply:
Good point. I'd be tempted to do that, and maybe also mention the debug log as an example so users don't wonder.

Reply:
Maybe also s/quality/quality and recency/

Reply:
Thanks, done.